cinema/models.py

```diff
@@ -35,12 +38,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(movie: "Movie", filename: str) -> pathlib.Path:
```

Comment:
The `movie_image_path` function is correctly defined to generate a unique file path for movie images. However, ensure that the `upload/movies/` directory exists and is writable in your media root, or handle the creation of this directory programmatically.